Procfile.v2

```diff
@@ -0,0 +1,6 @@
+# Use goreman to run `go get github.com/mattn/goreman`
```

Comment:
why do we need a v2 version?

Reply:
https://github.com/coreos/etcd/blob/master/Documentation/v2/proxy.md#proxy still reference proxy Procfile.